excellence-fee/admin/insert_fee.php

```diff
@@ -0,0 +1,72 @@
+<?php
+require_once('../../../../wp-load.php');
+
+global $wpdb;
+
+
+if(isset($_REQUEST['fee_value'])){
+	$fee=$_POST['fee_value'];
+	
```

Comment:
@Shraddha1596 why REQUESt and POST variable used same time, explain and correct it as well